catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/QueryOperations.java

```diff
@@ -340,6 +353,16 @@ boolean includesLocalSources(Set<String> sourceIds) {
                 || sourceIds.contains(null));
     }
 
+    private static Integer determineAndRetrieveMaxPageSize() {
+        try {
```

Comment:
I can't wait until I have the Apache Commons library memorized like you! :smile: Fixed